core/domain/interaction_jobs_one_off.py

```diff
@@ -152,6 +152,43 @@ def reduce(key, values):
         yield (key, values)
 
 
+class MultipleItemInteractionLtOneOffJob(
+        jobs.BaseMapReduceOneOffJobManager):
+    """Job that produces a list of (exploration ID, length) pairs
+    that use multiple choice interaction or item selection interaction
+    and a list of (exploration ID, (exploration, length)) pairs
+    that use multiple choice interaction or item selection
+    interaction whose choices length is greater than 30.
+    """
```

Comment:
Please move these to the `map` and `reduce` functions. This doc can just say something like "Audit job that flags MultipleItemInteractions with too many choices".

Reply:
Done.